src/main/java/uk/gov/gchq/palisade/service/request/DataRequestResponse.java

```diff
@@ -44,79 +43,81 @@ public DataRequestResponse() {
     }
 
     public DataRequestResponse token(final String token) {
```

Comment:
this mutator method needs @generated annotation adding

Reply:
Done